File: trunk/iphone/CloudManager/EmailBean.m
/**
 * Copyright (c) {2003,2010} {openmobster@gmail.com} {individual contributors as indicated by the @authors tag}.
 * All rights reserved. This program and the accompanying materials
 * are made available under the terms of the Eclipse Public License v1.0
 * which accompanies this distribution, and is available at
 * http://www.eclipse.org/legal/epl-v10.html
 */

#import "EmailBean.h"


/**
 * 
 * @author openmobster@gmail.com
 */
@implementation EmailBean

@synthesize oid;
@synthesize from;
@synthesize to;
@synthesize subject;
@synthesize date;

-(void)dealloc
{
	[oid release];
	[from release];
	[to release];
	[subject release];
	[date release];
	[super dealloc];
}

+(id)withInit
{
	EmailBean *instance = [[EmailBean alloc] init];
	instance = [instance autorelease];
	return instance;
}

@end

File: tags/2.2-M2/trunk/cloud/dev-tools/sampleApps/asyncserviceapp/app-android/src/main/java/org/async/service/app/cloud/app/command/EmailBean.java
/**
 * Copyright (c) {2003,2011} {openmobster@gmail.com} {individual contributors as indicated by the @authors tag}.
 * All rights reserved. This program and the accompanying materials
 * are made available under the terms of the Eclipse Public License v1.0
 * which accompanies this distribution, and is available at
 * http://www.eclipse.org/legal/epl-v10.html
 */

package org.async.service.app.cloud.app.command;

/**
 * Simple Email POJO on the device side
 * 
 * @author openmobster@gmail.com
 */
public class EmailBean
{
	private String oid;
	private String from;
	private String to;
	private String subject;
	private String date;
	
	public EmailBean()
	{
		
	}

	public String getFrom()
	{
		return from;
	}

	public void setFrom(String from)
	{
		this.from = from;
	}

	public String getTo()
	{
		return to;
	}

	public void setTo(String to)
	{
		this.to = to;
	}

	public String getSubject()
	{
		return subject;
	}

	public void setSubject(String subject)
	{
		this.subject = subject;
	}

	public String getDate()
	{
		return date;
	}

	public void setDate(String date)
	{
		this.date = date;
	}
	
	public String getOid()
	{
		return oid;
	}

	public void setOid(String oid)
	{
		this.oid = oid;
	}
}
